lib/perl/Genome/Model/Tools/Joinx/VcfMerge.pm

```diff
@@ -168,6 +173,9 @@ sub _resolve_flags {
     if ($self->merge_samples) {
         $flags .= " -s";
     }
+    if ($self->allow_same_file) {
```

Comment:
As I got further in this PR, it looks like we are checking the version compatibility of this feature in the context of the  VariantReporting framework in: [Genome/VariantReporting/Suite/Vep/Run.pm](https://github.com/genome/genome/pull/672/files#diff-6b6f65490e0c8fad06cc6756830f83d0R85) so it is being checked somewhere. It does maybe seem like knowledge of joinx feature compatibility should live in the joinx tools rather than in a VariantReporting/Vep module.